sos/report/plugins/vdo.py

```diff
@@ -28,6 +28,18 @@ def setup(self):
         vdos = self.collect_cmd_output('vdo list --all')
         for vdo in vdos['output'].splitlines():
             self.add_cmd_output(f"vdo status -n {vdo}")
-        self.add_cmd_output('vdostats --human-readable')
+        self.add_cmd_output([
+            'vdostats --human-readable',
+            'vdostats --verbose',
+        ])
+        vdo_cols = 'vdo_deduplication,vdo_index_state'
+        lvs_opts = '-a -o +'
+        lvm_vdos = self.collect_cmd_output(f"lvs {lvs_opts}{vdo_cols}")
+        for vdo in lvm_vdos['output'].splitlines():
```

Comment:
What if the `lvs` command would raise an error - is it safe to parse its output like this? (we often guard `'output'` under `if lvm_vdos['status'] == 0:` - but here I see this plugin lacks this check (cf. lines 28-29).

Reply:
I'll do this, sorry I missed the check.